src/notation/view/widgets/measureproperties.ui

```diff
@@ -214,6 +214,11 @@
               <string>64</string>
              </property>
             </item>
+            <item>
+             <property name="text">
+              <string>128</string>
+             </property>
+            </item>
```

Comment:
Isn't this going to be replaced with a qml version?

Reply:
Maybe, it isn't yet though, as far as I can tell